commands/register.py

```diff
@@ -0,0 +1,45 @@
+import nextcord
+from nextcord.ext import commands
+import requests
+import os
+
+headers={"X-Riot-Token": str(os.getenv("RIOT_KEY")),
+                                    "User-Agent": "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/118.0.0.0 Safari/537.36",
```

Comment:
Don't know why this is indented so heavily. Should probably be inline with X-Riot-Token

Reply:
Done